core/events/events_comment_resize.ts

```diff
@@ -0,0 +1,176 @@
+/**
+ * @license
+ * Copyright 2024 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * Class for comment resize event.
+ *
+ * @class
```

Comment:
I don't think `@class` is a valid TSDoc tag, can you remove?

Reply:
Done, I removed it from the two new events I added. There are many other files with this tag, but I imagine cleaning those up is outside the scope of this PR.